docs/index.html

```diff
@@ -93,8 +93,7 @@ <h1 class="desc-h1">Nighthawk and Meshery</h1>
       </div>
     <div class="nighthawk-meshery-div">
       <div class="container description-div description-div-grpc">
-        <div class="custom-desc-div">
-    
+        <div class="">
```

Comment:
```suggestion
        <div>
```

Reply:
@akshitarora921 Could you please make the small change as suggested above? Thanks for your contribution 🎉🚀

Reply:
@adithyaakrishna  Done